src/landing/search_bar.rs

```diff
@@ -195,9 +208,14 @@ impl WidgetMatchEvent for SearchBar {
         }
 
         if let Some(_) = input.changed(actions) {
+            x_button.set_visible(true);
```

Comment:
Right now, after the user types something for the first time, the X button stays visible forever.

Reply:
The `changed` method returns an Option<String> with the contents of the `TextInput`, you can check if the value is empty to hide the button, something like this:
```rs
if let Some(value) = input.changed(actions) {
     if value.is_empty() {
          x_button.set_visible(false);
     } else {
         x_button.set_visible(true);
    }
    ...
}
```

Also, I think we should me the icon a bit smaller and lighter color, right now it draws too much attention.

Reply:
I fix that case by adding that logic. Thank you very much.

> The `changed` method returns an Option with the contents of the `TextInput`, you can check if the value is empty to hide the button, something like this:
> 
> ```rust
> if let Some(value) = input.changed(actions) {
>      if value.is_empty() {
>           x_button.set_visible(false);
>      } else {
>          x_button.set_visible(true);
>     }
>     ...
> }
> ```
> 
> Also, I think we should me the icon a bit smaller and lighter color, right now it draws too much attention.